src/transformers/modeling_flax_utils.py

```diff
@@ -416,7 +416,8 @@ def save_pretrained(self, save_directory: Union[str, os.PathLike], push_to_hub=F
         # save model
         output_model_file = os.path.join(save_directory, FLAX_WEIGHTS_NAME)
         with open(output_model_file, "wb") as f:
-            model_bytes = to_bytes(self.params)
```

Comment:
this is required to allows weight saving in distributed mode - see `run_mlm_flax.py`

Reply:
Like the state_dict argument in the PyTorch equivalent, completely fine by me!